ironfish/src/utils/currency.ts

```diff
@@ -80,6 +75,28 @@ export class CurrencyUtils {
     }
   }
 
+  /**
+   * Renders values for human-readable purposes:
+   * - Renders $IRON in the major denomination, with 8 decimal places
+   * - If a custom asset, and `decimals` is provided, it will render the custom
+   *     asset in the major denomination with the decimal places
+   * - If a custom asset, and `decimals` is not provided, it will render the
+   *     custom asset in the minor denomination with no decimal places
+   */
+  static minorToMajor(
```

Comment:
Would you mind adding a few tests for this? Doesn't have to be crazy since this is a fairly light function that mostly rides on other tested functions - mostly just a way to provide some better visibility into SDK changes if they have to be altered etc

Reply:
Yep, will add